core/templates/dev/head/components/common-layout-directives/common-elements/background-banner.directive.ts

```diff
@@ -31,13 +31,15 @@ angular.module('oppia').directive('backgroundBanner', [
       controller: [
         function() {
           var ctrl = this;
+          this.$onInit = function() {
           var possibleBannerFilenames = [
             'bannerA.svg', 'bannerB.svg', 'bannerC.svg', 'bannerD.svg'];
           var bannerImageFilename = possibleBannerFilenames[
             Math.floor(Math.random() * possibleBannerFilenames.length)];
           ctrl.bannerImageFileUrl = UrlInterpolationService.getStaticImageUrl(
             '/background/' + bannerImageFilename);
         }
+      }
       ]
```

Comment:
Move this to previous line. Ditto elsewhere when you see a "}" directly above a (i.e. in the same column as a) "]".

In general, the closing brace/bracket should be at the same level as the opening one.

Reply:
Done everywhere